demos/src/main.scss

```diff
@@ -1 +1,13 @@
+$o-header-is-silent: false;
+
 @import "../../main";
+
+body {
+	margin: 0;
+	font-family: "MetricWeb";
+}
+
+.core .o--if-js,
+.enhanced .o--if-no-js {
+	display: none !important;
+}
```

Comment:
Oh, true, it adds `body { margin: 0; } .demo-js .o--if-nojs { display: none; }`; Should I change .demo-js to enhanced?